AirCasting/CreateSessionViews/TurnOnLocationView.swift

```diff
@@ -0,0 +1,108 @@
+// Created by Lunar on 27/07/2021.
+//
+
+import AirCastingStyling
+import CoreBluetooth
+import SwiftUI
+
+struct TurnOnLocationView: View {
+    @State private var isPowerABLinkActive = false
+    @State private var showAlert = false
+    @State private var isTurnBluetoothOnLinkActive = false
+    @State private var isMobileLinkActive = false
+    @EnvironmentObject var settingsRedirection: DefaultSettingsRedirection
+    @Binding var creatingSessionFlowContinues: Bool
+    @EnvironmentObject var bluetoothManager: BluetoothManager
+    @EnvironmentObject private var locationTracker: LocationTracker
+    @StateObject var sessionContext: CreateSessionContext
```

Comment:
all those dependencies should be inside a viewmodel hidden from View (manager, tracker, context, redirection)